pyvista/plotting/plotter.py

```diff
@@ -471,7 +488,7 @@ def theme(self) -> Theme:  # numpydoc ignore=RT01
         >>> import pyvista as pv
         >>> from pyvista import themes
         >>> pl = pv.Plotter()
-        >>> pl.theme = themes.DarkTheme()
+        >>> pl.theme = themes.DarkTheme()  # doctest: +IGNORE_WARNINGS
```

Comment:
`pytest-doctestplus` isn't working because it isn't enabled in the pytest options (or invoked as a flag on the invocation, etc.).  But, adding this will fundamentally change our doc test for the whole package.  This would have potential unforeseen issues.

I have two ideas to fix this issue.  Since it is deprecated and we have tests for that, it isn't that crucial to have an additional test in the documentation.
1) remove the example entirely.
2) skip the example using `+SKIP`